app/Notifications/OrganizationInvite.php

```diff
@@ -55,11 +58,21 @@ public function via($notifiable)
      */
     public function toMail($notifiable)
     {
+        if ($this->page == 'register') {
+            $url = $this->pageUrl . '/' . $this->page . '/' . $this->organization->domain .'?email='.$this->user_email;
+            $subject = 'Invitation to join the organization';
+            $caption = 'Join the Organization';
```

Comment:
Formatting issue; need space before and after concatenating operator.
Also, I think === comparison should be used.

Reply:
formatting and comparison issue fixed.